ontology/ontoneo.md

```diff
@@ -2,32 +2,49 @@
 layout: ontology_detail
 id: ontoneo
 title: Obstetric and Neonatal Ontology
-browsers:
-- title: BioPortal Browser
-  label: BioPortal
-  url: https://bioportal.bioontology.org/ontologies/ONTONEO
-build:
-  source_url: http://purl.obolibrary.org/obo/ontoneo/ontoneo.owl
 contact:
   email: fernanda.farinelli@gmail.com
   github: FernandaFarinelli
   label: Fernanda Farinelli
   orcid: 0000-0003-2338-8872
 description: The Obstetric and Neonatal Ontology is a structured controlled vocabulary to provide a representation of the data from electronic health records (EHRs) involved in the care of the pregnant woman, and of her baby.
-domain: health
-homepage: ontoneo.com
 license:
-  label: CC BY 3.0
-  url: http://creativecommons.org/licenses/by/3.0/
+  label: CC BY 4.0
+  url: https://creativecommons.org/licenses/by/4.0/
+domain: health
+homepage: https://ontoneo.com/
 mailing_list: http://groups.google.com/group/ontoneo-discuss
 preferredPrefix: ONTONEO
 products:
 - id: ontoneo.owl
+  title: ONTONEO
+  description: The full version of ONTONEO in OWL format
+build:
+  source_url: https://raw.githubusercontent.com/ontoneo-project/Ontoneo/master/ontoneo.owl
 repository: https://github.com/ontoneo-project/Ontoneo
-tags:
-- biomedical
 tracker: https://github.com/ontoneo-project/Ontoneo/issues
+browsers:
+  - label: BioPortal
+    title: BioPortal Browser
+    url: https://bioportal.bioontology.org/ontologies/ONTONEO
+  - label: OntoBee
+    title: OntoBee Browser
+    url: https://ontobee.org/ontology/ONTONEO
+publications:
+  - id: https://doi.org/10.5281/zenodo.17429771
+    title: "OntONeo: The Obstetric and Neonatal Ontology"
+  - id: https://doi.org/10.22477/ISKO.25.99
```

Comment:
Unfortunately, this DOI leads to a page that says "DOI Not Found". Is it possible there was a typo?

Reply:
In fact, this is the DOI for the conference proceedings. I create another one with Zenodo.